composer.json

```diff
@@ -23,7 +23,7 @@
         "php": ">=5.6"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5",
+        "phpunit/phpunit": "^5.7 || ^6.4 || ^7.0",
```

Comment:
What's the point of this and not only `^7.0`?

Reply:
We still support PHP 5.6

Reply:
But we could drop 5.x from the requirements.